<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>586</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>771</width>
    <height>586</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Rechenknecht</string>
  </property>
  <property name="windowIcon">
   <iconset theme="calc.png" resource="Resources/res.qrc">
    <normaloff>:/Res/calc_trans.png</normaloff>:/Res/calc_trans.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>650</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_bin">
         <property name="text">
          <string>Bin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_oct">
         <property name="text">
          <string>Oct</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_dec">
         <property name="text">
          <string>Dec</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_hex">
         <property name="text">
          <string>Hex</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="3" column="6">
        <widget class="QPushButton" name="pushButton_prozent">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_7">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="Line" name="line">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="pushButton_6">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="pushButton_1">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_B">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item row="9" column="5">
        <widget class="QPushButton" name="pushButton_multiplication">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QPushButton" name="pushButton_vorzeichen_wechsel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>+/-</string>
         </property>
        </widget>
       </item>
       <item row="5" column="6">
        <widget class="QPushButton" name="pushButton_left_bracket">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>(</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButton_F">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>F</string>
         </property>
        </widget>
       </item>
       <item row="9" column="4">
        <widget class="Line" name="line_5">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_E">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>E</string>
         </property>
        </widget>
       </item>
       <item row="10" column="5">
        <widget class="QPushButton" name="pushButton_division">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QPushButton" name="pushButton_plus">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="pushButton_C">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="Line" name="line_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QPushButton" name="pushButton_0">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="10" column="4">
        <widget class="Line" name="line_6">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="pushButton_5">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="10" column="6">
        <widget class="QPushButton" name="pushButton_pi">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>ùúã</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_D">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>D</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="pushButton_wurzel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>x‚àöy</string>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="QPushButton" name="pushButton_minus">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QPushButton" name="pushButton_equal">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="Line" name="line_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="pushButton_9">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QPushButton" name="pushButton_clear">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>CE</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="Line" name="line_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="6">
        <widget class="QPushButton" name="pushButton_e">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>e</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QPushButton" name="pushButton_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_A">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="pushButton_8">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="pushButton_hoch">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>x^y</string>
         </property>
        </widget>
       </item>
       <item row="4" column="6">
        <widget class="QPushButton" name="pushButton_right_bracket">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="Rechenknecht">
    <property name="title">
     <string>Rechenknecht</string>
    </property>
    <addaction name="Beenden"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionEntwickler_kontaktieren"/>
   </widget>
   <addaction name="Rechenknecht"/>
   <addaction name="menuHilfe"/>
  </widget>
  <action name="Beenden">
   <property name="text">
    <string>Beenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionEntwickler_kontaktieren">
   <property name="text">
    <string>Entwickler kontaktieren</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_equal</tabstop>
  <tabstop>pushButton_clear</tabstop>
  <tabstop>pushButton_plus</tabstop>
  <tabstop>pushButton_right_bracket</tabstop>
  <tabstop>pushButton_minus</tabstop>
  <tabstop>pushButton_left_bracket</tabstop>
  <tabstop>pushButton_multiplication</tabstop>
  <tabstop>pushButton_division</tabstop>
  <tabstop>pushButton_pi</tabstop>
 </tabstops>
 <resources>
  <include location="Resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
